rca/api_content/templates/api_content/api_content_admin.html

```diff
@@ -0,0 +1,32 @@
+{% extends "wagtailadmin/base.html" %}
+{% load wagtailadmin_tags static i18n %}
+{% block titletag %}{% trans "Dashboard" %}{% endblock %}
+{% block bodyclass %}homepage{% endblock %}
+
+
+{% block content %}
+    <script>
+    function clear_cache() {
+        $.get('/admin/api-cache/clear/', function (data) {
+            $("#cache_clear_button").removeClass("button-longrunning-active")
```

Comment:
No need to use jQuery IMO.

```js
async function clearCache() {
  try {
    const response = await fetch("/admin/api-cache/clear/");
    if (!response.ok) {
      throw Error(`Unexpected HTTP code ${response.status}`);
    }
    document.getElementById('cache_clear_button').classList.remove('button-longrunning-active');
    document.getElementById('cache_clear_button').disabled = true;
  } catch(e) {
    // error handling.
  }
}
```
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API